### PR TITLE
Fix iGPU CMakeFile tags

### PR DESCRIPTION
**Pull request by kthui**

### CODE REVIEW

File: Dockerfile.QA
Comment (nv-kmcgill53) [inline Dockerfile.QA:1]: @kthui Can you confirm my syntax is correct in this file

Comment (kthui) [reply Dockerfile.QA:1]: Pre-commit should be fixed by [Pre-commit fix](https://github.com/triton-inference-server/server/pull/6695/commits/83183e71a0346af37681f24ac0ea6b7fb4e8e221)

File: src/test/CMakeLists.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ find_package(GTest REQUIRED)
 #
 # caffe2plan
 #
-if(${TRITON_ENABLE_TENSORRT})
+if(${TRITON_ENABLE_TENSORRT} AND NOT ${TRITON_IGPU_BUILD})
```
Comment (mc-nv) [inline src/test/CMakeLists.txt:37]: Are we going to fee our `build.py` with extra CMake argument. 
I wasn't able from glance find it.

Comment (kthui) [reply src/test/CMakeLists.txt:37]: I think @nv-kmcgill53 is right that we cannot build this part as it involves some peculiar linking. It used to be there I think.

Comment (nv-kmcgill53) [reply src/test/CMakeLists.txt:37]: @mc-nv We are currently adding this to the build. [See here](https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/blob/3fe14c296226d262d5299c516f589ba5bd564a20/.gitlab-ci.yml#L1126)

**Pull request merged**